proto/lekko/client/v1beta1/configuration_service.proto

```diff
@@ -93,6 +93,12 @@ message GetProtoValueRequest {
 
 message GetProtoValueResponse {
   google.protobuf.Any value = 1;
+  Any value_new = 2;
```

Comment:
nit: `value_v2`? "new" will become a bit confusing when it will be the only option that is used :)

Reply:
updated to v2. i'll do another pass of renaming this field once the existing field is fully deprecated as field renames are backwards compatible